### PR TITLE
Add a simple README file

### PR DESCRIPTION
**Pull request by red-quinoa**
Add some basic information so that the page is not empty

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Rearrange
+=========
+
+This module is used for rearranging names.
+Turns "LastName, FirstName" into "FirstName LastName"
```
Comment (blue-kale) [inline README.md]: This sentence is missing a period at the end.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+
+# Example
```
Comment (blue-kale) [inline README.md]: Please make this ## so that it's a smaller title.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
...
+
+Calling `rearrange_name("Turing, Alan")` will return `Alan Turing`
```
Comment (blue-kale) [inline README.md]: This is cool, but please add a couple more examples.

File: README.md (hunk 1 above)
Comment (red-quinoa) [reply README.md]: Done

File: README.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Rearrange
```
Comment (blue-kale) [inline README.md]: Use #

**Pull request closed without merging**